torch/csrc/inductor/aot_runtime/model_container.h

```diff
@@ -150,14 +150,13 @@ class AOTInductorModelContainer {
           0 // device index, should read it from cudaStream_t?
           ));
       constants_->emplace(
-          std::move(name),
-          std::move(steal_tensor_handle_from_raw_to_raii(tensor_handle)));
+          std::move(name), std::move(UniqueAtenTensorHandle(tensor_handle)));
```

Comment:
The new name is worse because you lost the idea that this is causing a steal. It's not completely out of line because `unique_ptr` and `shared_ptr` constructors steal their raw pointer inputs but I'd certainly feel better with, e.g., an `enum class` sigil in front like `SharedAtenTensorHandle(SharedAtenTensorHandle::STEAL, tensor_handle)`

Reply:
This class always steals, so not sure if we want to define an enum here. I can change it back to RAIIAtenTensorHandle since that usually implies the handle is managed by this class, if you think that's better.

Reply:
I know it always steals, the point is to make it obvious to users that it always steals. In particular, you didn't write something like `unique_ptr(new T())` which is a pretty clear indicator that stealing is going on. The point of the enum isn't to let you pick between different choices (although you can certainly use it to do that), it's to force people to type STEAL. You could also do it with a static method but then you'd need a private constructor that the static method hits.